components/atom/button/src/index.scss

```diff
@@ -175,6 +176,7 @@ $icon-stroke-color: currentColor !default;
   &-group {
     margin-left: -1px;
     margin-right: 0;
+    padding: 0 $p-atom-button-group;
```

Comment:
I suggest one of these options:
- `padding: $p-atom-button-group;`
- `padding: 0 $p-atom-button-group-sides ;`
- `padding: 0 $p-atom-button-group-LeftRight;`

instead of 
`padding: 0 $p-atom-button-group;`

because `padding` defines the padding for the 4 sides